payu/models/cesm_cmeps.py

```diff
@@ -263,6 +263,11 @@ def _setup_checks(self):
         return True
 
     def archive(self):
+
+        # Move any the MOM_parameter_docs output back into the control repo and commit it for documentation
+        if 'mom' in self.components.values() :
+            mom6_save_docs_files(self)
```

Comment:
So it's necessary to call this explicitly because the `cesm_cmeps` driver doesn't directly call the `mom6.archive` function?